src/styles/blocks/footer.scss

```diff
@@ -0,0 +1,14 @@
+.footer {
+  height: 200px;
+  background-image: url(../images/photos/footer.png);
```

Comment:
The same point is about this image size.

Reply:
fixed